src/api/helpers/decrypt/index.ts

```diff
@@ -0,0 +1,20 @@
+import crypto from 'crypto';
+import dotenv from 'dotenv';
+import { FINANCIAL_DETAILS } from '../../constants';
+import generateKey from '../encrypt/generate-key';
+import { FinancialDetailsEncryption } from '../../types';
+
+dotenv.config();
+
+const { ENCRYPTION_METHOD, ENCODING, STRING_ENCODING, OUTPUT_ENCODING } = FINANCIAL_DETAILS.ENCRYPTION.CIPHER;
+
+const key = generateKey();
+
+const decrypt = (dataToDecrypt: FinancialDetailsEncryption) => {
+  const buff = Buffer.from(dataToDecrypt.value, STRING_ENCODING);
```

Comment:
Just noticed can you do the following please to keep it consistent.

```javascript
const { value, iv } = dataToDecrypt;
```

Reply:
will be done in next PR with refactor